### ISSUE TITLE
Use a tracker that is respectfull of your user's privacy (Matomo?)

### ISSUE THREAD

**Issue by Tbaut** [blockscout/blockscout#1274]
Hello there, Thibaut from Parity.
We [recently moved](https://github.com/paritytech/fether/issues/311) from Etherscan to Blockscout with our light client based wallet to enhance our user' privacy, and because Blockscout is open source and we want to promote it because we love what you guys do :heart: 

There's still one little thing that is bothering me, it's the use of Google Analytics (and Google tag manager) and its consequences for your user's privacy. This video explains it well: https://slideslive.com/38911970/plugging-the-metadata-leaks-in-the-ethereum-ecosystem

TL;DR Such information as the link between an IP to an Eth address is *very* tricky to handle, and it better land in the right hands. I prefer yours than google's and that's also why I would suggest to use open source tools such as [~~Piwik~~ Matomo](https://matomo.org/) instead, where the data you gather is not shared with anyone.

Other than that, thank you for your work!

**Comment by igorbarinov** [comment]
Ok, google analytics is not integrated into BlockScout. It's injected in Cloudflare. I think we can replace it with another one, e.g. Matomo.

**Comment by SiddigZeidan** [comment]
Good point @Tbaut , and good to see that the light base wallet has moved to BlockScout! :-D

**Comment by igorbarinov** [comment]
Hey @Tbaut 
- we added Matomo last week and will test it for a week. After that, we'll remove GA from mainnet.
- the team made major speedup for the UI

we are still indexing the full node for mainnet. ETA is 11 days.
```
Indexed Tokens: 205,773,824
Remaining Tokens: 40,625,165
Daily Percent: 1.35%
Tokens Indexed Last 24 hrs:  3,342,441
ETA: 11.7 days
```

**Comment by Tbaut** [comment]
This is a great Christmas present, thanks and congrats!

**Comment by igorbarinov** [comment]
Sorry about that. I'll open it again.
Andrew, please don't close issues without retesting.

**Comment by acravenho** [comment]
Google analytics has been removed from Cloudflare.
